src/main/java/com/example/finance/entities/User.java

```diff
@@ -0,0 +1,18 @@
+package com.example.finance.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import lombok.Data;
+
+@Entity // utilizo para indicar ao JPA que essa classe é uma entidade.
+@Data // utilizo para gerar os métodos getters e setters, toString, equals e hashCode.
+public class User {
+
+  @Id
+  @GeneratedValue
```

Comment:
```suggestion
  @GeneratedValue(strategy = generationtype.AUTO)
```
Aqui talvez seria bom definir o tipo de strategy do Id